src/Opdex.Platform.WebApi/Models/Responses/Tokens/MinedTokenDistributionScheduleResponseModel.cs

```diff
@@ -0,0 +1,29 @@
+using Opdex.Platform.Common.Models;
+
+namespace Opdex.Platform.WebApi.Models.Responses.Tokens;
+
+/// <summary>
+/// Distribution schedule details.
+/// </summary>
+public class MinedTokenDistributionScheduleResponseModel
+{
+    /// <summary>
+    /// Address of the vault contract
+    /// </summary>
+    public Address Vault { get; set; }
+
+    /// <summary>
+    /// Address of the mining governance contract
+    /// </summary>
+    public Address MiningGovernance { get; set; }
+
+    /// <summary>
+    /// Block number at which the next distribution becomes available
+    /// </summary>
+    public ulong NextDistributionBlock { get; set; }
+
+    /// <summary>
+    /// Previous distribution details
+    /// </summary>
+    public MinedTokenDistributionItemResponseModel Previous { get; set; }
```

Comment:
Thinking `Distributions` might be clearer than previous, even though it is within a `Distribution` property of a token. 

Just thinking really, `Previous' isn't really descriptive. Maybe "HistoricalDistributions", "Distributions", "PreviousDistributions", "Mints" etc...not sure 🤔

Reply:
I've updated it to `History` now it's returning every historical distribution. Since it's within a `distribution` object, I think this is explicit enough.

```json
"distribution": {
    "vault": "PAe1RRxnRVZtbS83XQ4soyjwJUDSjaJAKZ",
    "miningGovernance": "PDqHiacQ6vb3h48EBp8tKqUyvzK1NEG1KQ",
    "nextDistributionBlock": 176646,
    "history": [
        {
            "vault": "100000000.00000000",
            "miningGovernance": "300000000.00000000",
            "block": 68646
        }
    ]
}
```